phpBB/phpbb/search/fulltext_sphinx.php

```diff
@@ -430,22 +435,27 @@ protected function config_generate()
 	*/
 	public function split_keywords(&$keywords, $terms)
 	{
+		// Keep quotes and new lines
+		$keywords = str_replace(array('&quot;', "\n"), array('"', ' '), trim($keywords));
```

Comment:
If possible, please use the new array syntax `[]` instead of the old `array()`.
For here and other edited / added code below.